<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>576</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="console">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>控制台</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>561</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>360</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>启动</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="stopButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>420</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>20</y>
        <width>180</width>
        <height>220</height>
       </rect>
      </property>
      <property name="title">
       <string>服务端设置</string>
      </property>
      <widget class="QLabel" name="serverip">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;监听IP：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="serverport">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;端口：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ipEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>160</width>
         <height>30</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>任意</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="portEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>160</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>8899</string>
       </property>
       <property name="placeholderText">
        <string>不输入为任意</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="signature">
     <attribute name="title">
      <string>协同签名数据库</string>
     </attribute>
     <widget class="QTableWidget" name="signKeyTableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>69</y>
        <width>760</width>
        <height>451</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="rowCount">
       <number>14</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>639</number>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>33</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>36</number>
      </attribute>
      <row>
       <property name="text">
        <string>公钥</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string>客户端私钥</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>客户端公钥</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string>服务端私钥</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>服务端公钥</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string>椭圆曲线参数a</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>椭圆曲线参数b</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>椭圆曲线参数p</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>椭圆曲线基点</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string>更新时间</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>值</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="usernameEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>431</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请输入要查询的用户名</string>
      </property>
     </widget>
     <widget class="QLabel" name="username">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>20</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;用户名：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchSignKeyButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>20</y>
        <width>104</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="encryption">
     <attribute name="title">
      <string>密钥分割数据库</string>
     </attribute>
     <widget class="QLineEdit" name="uidEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>431</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请输入要查询的用户名1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchEncryptKeyButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>60</y>
        <width>104</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLabel" name="uid">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>20</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;用户名1：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="encryptKeyTableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>119</y>
        <width>760</width>
        <height>401</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="rowCount">
       <number>13</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>656</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>33</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>36</number>
      </attribute>
      <row>
       <property name="text">
        <string>模数p</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>多项式参数a1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>多项式参数a2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>密钥</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>用户1密钥对</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string>用户2密钥对</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string>服务端密钥对1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string>服务端密钥对2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string>更新时间</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>值</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="touidEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>431</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请输入要查询的用户名2</string>
      </property>
     </widget>
     <widget class="QLabel" name="touid">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>60</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;用户名2：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
